src/editor/components/components/AddLayerPanel/createLayerFunctions.js

```diff
@@ -172,22 +174,45 @@ export function createCustomModel() {
   }
 }
 
-export function createPrimitiveGeometry() {
+export function createPrimitiveGeometry(position) {
   const definition = {
-    'data-layer-name': 'Plane Geometry • Traffic Circle Asphalt',
+    'data-layer-name': 'Geometry • Traffic Circle Asphalt',
     components: {
-      geometry: 'primitive: circle; radius: 50;',
+      position: position ?? '0 0 0',
+      geometry: 'primitive: circle; radius: 15;',
       rotation: '-90 -90 0',
       material: 'src: #asphalt-texture; repeat: 5 5;'
     }
   };
   AFRAME.INSPECTOR.execute('entitycreate', definition);
 }
 
-export function createIntersection() {
+export function createImageEntity(position) {
+  // This component accepts a svgString and creates a new entity with geometry extruded
+  // from the svg and applies the default mixin material grass.
+  const imagePath = prompt(
+    'Please enter an image path that is publicly accessible on the web and starts with https://',
+    `https://assets.3dstreet.app/images/signs/Sign-Speed-30kph-Kiritimati.png`
+  );
+  if (imagePath && imagePath !== '') {
+    const definition = {
+      element: 'a-entity',
+      components: {
+        position: position ?? '0 0 0', // TODO: How to override only the height (y) value? We don't want the sign in the ground
```

Comment:
you can construct a new position string or an object {x: 0, y: 1, z: 0} here

Reply:
thanks, i'll worry about that one later to just get this out